### ISSUE TITLE
New release with pytorch 1.12 support?

### ISSUE THREAD

**Issue by craigcitro** [fastai/fastai#3730]
tl;dr: fastai 2.7.5 still requires `torch<1.12`.

I was looking to [update torch in Colab](https://github.com/googlecolab/colabtools/issues/2893), but hit trouble due to a version conflict: `fastai==2.7.5` still requires `torch<1.12`. It looks like https://github.com/fastai/fastai/commit/363156e5b21ef1b7f611d47d776870e0d1c7d339 didn't quite make the 2.7.5 cut, though the release notes suggest torch support 1.12 was intended to make the cut.

Any chance at a 2.7.6 that includes 363156e5b21ef1b7f611d47d776870e0d1c7d339?

Thanks!

Please confirm you have the latest versions of fastai, fastcore, and nbdev prior to reporting a bug (delete one): YES

**Comment by craigcitro** [comment]
Looks like this got fixed -- thanks!
